packages/screens/AllProjectAdministrationDash/AllProjectAdministrationDashScreen.tsx

```diff
@@ -0,0 +1,370 @@
+import React, { ReactNode, useState } from "react";
```

Comment:
Maybe you can split this file to have components in specific files ?

Reply:
Fixed, PR updated